### ISSUE TITLE
[NEW TOOL] QR Code decoder

### ISSUE THREAD

**Issue by SDBdevelopment** [CorentinTh/it-tools#388]
A browser extension to decode (use backup function) you QR Codes.

https://github.com/Authenticator-Extension/Authenticator

Thank you.

**Comment by TCB13** [comment]
You're suggesting two different things.

1. Ability to decode a QR code, already suggested in here: https://github.com/CorentinTh/it-tools/issues/275#issuecomment-1483552639
2. Generate the "2-step" codes, already implemented here https://it-tools.tech/otp-generator of you're willing to decode the QR code with another tool.

I suggest you change the title of the issue to "OTP code generator: Upload QR Code".

**Comment by nomandera** [comment]
Another example of this this could be done 

```
zbarimg -q it-toolsqr.png 
QR-Code:otpauth://totp/IT-Tools:demo-user?issuer=IT-Tools&secret=W3TYSQVLBN6BHOUR&algorithm=SHA1&digits=6&period=30
```

**Comment by sharevb** [comment]
Hi @SDBdevelopment, @nomandera, @TCB13  and @CorentinTh, QRCode decoding in https://github.com/CorentinTh/it-tools/pull/914

**Comment by SDBdevelopment** [comment]
Hi @sharevb it works great! Thank you!

**Comment by sharevb** [comment]
Hi, for people interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Comment by sharevb** [comment]
Hi @SDBdevelopment and for people interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Comment by sharevb** [comment]
Hi @SDBdevelopment, @nomandera, @TCB13 if you are interested in an up to date version of it-tools, with many improvements, new tools, and bug fixes, as this repo is almost no more maintained, I made a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Comment by TCB13** [comment]
Yeah this project seems to be a bit... dead. Lot of PRs, nothing happens. Thank you.

**Comment by sharevb** [comment]
Hi @TCB13, yes, so I made a fork to not have been things for nothing...
